### PR TITLE
feat(am): added image api

### LINKED ISSUE #28: Add image to asset
**Is your feature request related to a problem? Please describe.**
User would like to upload image to asset.

**Describe the solution you'd like**
Added endpoints for upload, delete and patch image.

**Additional context**
The functionality is the same as for user-management.

### PR DESCRIPTION
**Pull request by anatoliynovo**
Quick checklist for code changes:
- [x] Created a new branch based on `dev`
- [x] This PR will be submitted back in `dev`
- [x] All commits correspond to our [convention](https://fraunhoferisst.github.io/diva-docs/dev-docs/contribution/commits.html)
- [x] The code linted (we need to check lerna requirement injection)
- [ ] All checks passed
- [x] README's updated (if configuration, behavior changed)
- [x] OpenAPI specs updated (if API changed)
- [x] Versions for changed components incremented (in the component specific file and in corresponding `docker-compose.yml`)
- [x] Any breaking changes clearly communicated to the core team and do not imply system dysfunction
- [x] [Documentation](https://github.com/FraunhoferISST/diva-docs) update proposed (if configuration, behavior, structure, processes changed)
Closes #28

### CODE REVIEW

**Comment by DaTebe** [comment]
@setaman you said something was off here?

**Comment by setaman** [comment]
@DaTebe 
yeah, need a clarification from @anatoliynovo on [this](https://github.com/FraunhoferISST/diva/pull/29/files/bd4a6c3fceadeb634c0f40f388b298d325281d66#diff-4eccfdb4f7301f7edf11a5a1d7e943cb3927407b03be3fa62568b81decbd994cR87)

**Comment by DaTebe** [comment]
Should be more like this: https://github.com/FraunhoferISST/diva/blob/23856f7287b25518153749da8ea6dd665f96ddfc/core/services/user-management/controllers/UserImagesController.js#L29

?

**Comment by setaman** [comment]
exactly, that's why i'am asking

**Comment by anatoliynovo** [comment]
my bad guys,  you're right. I once had issue with deleting, but in the end it was due to the version of mongo, and forgot to remove old code. I updated the branch like it supposed to be.

**Review by setaman — APPROVED** [review]

File: core/services/asset-management/services/AssetImagesService.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  }
+
+  async deleteImage(id) {
+    if (!(await imageExists(id))) {
+      throw imageNotFoundError;
+    }
+    const documents = await assetsMongoDbConnector.gfs
+      .find({ _id: id })
+      .toArray();
+
```
[the remark this answers is not in the document]
Comment (DaTebe) [reply core/services/asset-management/services/AssetImagesService.js]: @setaman is this question still to be answered? You already approved all changes.

[the remark this answers is not in the document]
Comment (setaman) [reply core/services/asset-management/services/AssetImagesService.js]: Merge it

**Pull request merged**